js/geo/coordinate.js

```diff
@@ -15,6 +15,7 @@ function Coordinate(column, row, zoom) {
     this.column = column;
     this.row = row;
     this.zoom = zoom;
+    this.posMatrix = null;
```

Comment:
I'm 👎 on caching this within `Coordinate`. The `posMatrix` is a function of coordinate, source, and transform. It would make more sense to cache within the `calculatePosMatrix` method itself.